JoyShockLibrary/JoyShock.cpp

```diff
@@ -470,7 +470,7 @@ class JoyShock {
 		unsigned char buf[0x40];
 		memset(buf, 0, 0x40);
 
-		uint8_t rumble_base[9] = { (++global_count) & 0xF, 0x00, 0x01, 0x40, 0x40, 0x00, 0x01, 0x40, 0x40 };
+		uint8_t rumble_base[9] = { std::uint8_t((++global_count) & 0xF), 0x00, 0x01, 0x40, 0x40, 0x00, 0x01, 0x40, 0x40 };
```

Comment:
This was needed because GCC and Clang don't allow narrowing conversions in initializer lists